worker/src/sms/services/twilio-client.class.ts

```diff
@@ -70,7 +70,7 @@ export default class TwilioClient {
     callbackUrl.username = username
     // encode password as the hash contains special characters
     callbackUrl.password = encodeURIComponent(hashedPwd)
-    callbackUrl.pathname = `${callbackUrl.pathname}/campaign/${campaignId}/message/${messageId}`
+    callbackUrl.pathname = `${callbackUrl.pathname}/${campaignId}/${messageId}`
```

Comment:
note to self: fix this, it's not backwards compatible. 
to make it backwards compatible, we need to first release a route 
postman.gov.sg/v1/callback/$campaignId/$messageId that redirects to callback.postman.gov.sg/v1/messages/campaign/$campaignId/message/$messageId